compass/meta.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<meta>
+  <info name="Extra Chat" type="script" description="Extra chat functionality and commands" author= "3marproof" version="1.0" />

```

Comment:
Why is this called "Extra Chat", and "Extra chat functionality and commands"? Just name it "Compass".